cmd/metrics-server/app/options/options.go

```diff
@@ -63,6 +63,23 @@ type Options struct {
 	DeprecatedCompletelyInsecureKubelet bool
 }
 
+func (o *Options) Validate() []error {
+	/*
+	Added this for issue 571
+	Using manual rules
+	*/
+	return_list_of_errors := []error{}
```

Comment:
```suggestion
	errors := []error{}
```

No need for too long names

Reply:
Okay